### PR TITLE
Project basic information update

### PR DESCRIPTION
**Pull request by dimasciput**
cadasta-project-update

### CODE REVIEW

File: cadasta/gui/tools/wizard/step_project_update02.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        return (
+            True,
+            ''
+        )
+
+    def get_next_step(self):
+        """Find the proper step when user clicks the Next button.
+
+        :returns: The step to be switched to
+        :rtype: WizardStep instance or None
```
Comment (timlinux) [inline cadasta/gui/tools/wizard/step_project_update02.py]: :rtype: should be an actual python type (or list of types separated by commas e.g.

WizardStep, None

**Comment by timlinux** [comment]
LGTM

File: cadasta/gui/tools/wizard/step_project_update02.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+                selected_item.setSelected(True)
+
+    def validate_step(self):
+        """Check if the step is valid.
+
+        :returns: Tuple of validation status and error message if any
+        :rtype: ( bool, str )
+        """
+        return (
```
Comment (ismailsunni) [inline cadasta/gui/tools/wizard/step_project_update02.py]: Is it correct, that it always returns True?

Comment (dimasciput) [reply cadasta/gui/tools/wizard/step_project_update02.py]: Yes for this step.

**Pull request merged**